### PR TITLE
Run CanvasUITests on Bitrise

### PR DESCRIPTION
**Pull request by armstrongnate**

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student

[MBL-12525](https://instructure.atlassian.net/browse/MBL-12525)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.17% | 95.17% | 0%
**Student** | 92.89% | 92.89% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.7% | 92.7% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Canvas/Canvas.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4833,6 +4833,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = B6333T4PXQ;
+				ENABLE_TESTABILITY = YES;
```
Comment (vanwagonet) [inline Canvas/Canvas.xcodeproj/project.pbxproj]: I don't think we want to do this.

File: Canvas/Canvas.xcodeproj/xcshareddata/xcschemes/Canvas.xcscheme [hunk 2]
```diff
@@ -23,7 +23,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (vanwagonet) [inline Canvas/Canvas.xcodeproj/xcshareddata/xcschemes/Canvas.xcscheme]: If we are in release mode, won't that hurt our ability to find out where failures were?

**Comment by armstrongnate** [comment]
@vanwagonet got the builds passing with this https://github.com/instructure/canvas-ios/pull/316

Thanks Andy! Ima close this.

**Pull request closed without merging**